sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -472,14 +474,24 @@ object CatalogColumnStat extends Logging {
   private val KEY_MAX_LEN = "maxLen"
   private val KEY_HISTOGRAM = "histogram"
 
+  val VERSION = 2
```

Comment:
@cloud-fan Should I introduce a SQL config for current version?

Reply:
we can't. Version is an internal property of CBO stats, which can only be increased when we change format.

Reply:
but do you agree to bump version up to 2? otherwise I cannot distinguish when to parse by using  system time zone or by `UTC`.